jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -440,23 +407,24 @@ static void write(Sink sink, boolean last, String utf8Content, Callback callback
         Content.Chunk EOF = ByteBufferChunk.EOF;
 
         /**
-         * <p>Creates a last/non-last Chunk with the given ByteBuffer.</p>
+         * <p>Creates a Chunk with the given ByteBuffer.</p>
+         * <p>The returned Chunk must be {@link #release() released}.</p>
          *
          * @param byteBuffer the ByteBuffer with the bytes of this Chunk
          * @param last whether the Chunk is the last one
          * @return a new Chunk
          */
         static Chunk from(ByteBuffer byteBuffer, boolean last)
         {
-            return new ByteBufferChunk(byteBuffer, last);
+            return new ByteBufferChunk.WithReferenceCount(byteBuffer, last);
```

Comment:
I don't think this one does need a reference count as there is no release action.  It can have a noop retain and a release that always returns false.